google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQuery.java

```diff
@@ -289,6 +289,10 @@ public static DatasetOption fields(DatasetField... fields) {
       return new DatasetOption(
           BigQueryRpc.Option.FIELDS, Helper.selector(DatasetField.REQUIRED_FIELDS, fields));
     }
+
+    public static DatasetOption accessPolicyVersion(Integer accessPolicyVersion) {
```

Comment:
Can we get a doc string? In particular, the user would need to understand what the accessPolicyVersion integer values mean. A link to an article or public documentation is sufficient.